lms/djangoapps/ora_staff_grader/mock/views.py

```diff
@@ -94,6 +94,11 @@ class UpdateGradeView(RetrieveAPIView):
     """ Submit a grade """
 
     def update_grade_data(self, submission, grade_data):
+        # this is static test data
+        grade_data['score'] = {
+            "pointsEarned": 70,
+            "pointsPossible": 100
+        }
```

Comment:
This should be in the serialized return instead of `update_grade_data`

Reply:
e.g. after `save_submission_update`:

```
save_submission_update(ora_location, submission)

# Add mock score data (until we add accurate score calculation)
grade_data = submission['gradeData']
grade_data['score'] =  {
    "pointsEarned": 70,
    "pointsPossible": 100
}

return ...
```

Reply:
it is exist in both place according to our source of truth.

Reply:
I mean, add the score to grade data like you're doing, just move it out of that function because it doesn't make sense to be there and adds side-effects which isn't great practice.

Reply:
^ And then change `return... 'gradeData': submission['gradeData'],` to include the raw `gradeData`:

```
        return Response({
            'gradeStatus': submission['gradeStatus'],
            'lockStatus': submission['lockStatus'],
            'gradeData': grade_data  # this should return the right data without side effects
        })
```